nuspec/Frends.Web.nuspec

```diff
@@ -18,8 +18,8 @@
     </frameworkAssemblies>
   </metadata>
   <files>
-	<file src="Frends.Web\bin\Release\Frends.Web.dll" target="lib\net452\Frends.Web.dll" />
-    <file src="Frends.Web\bin\Release\Frends.Web.XML" target="Frends.Web.XML"/>
-    <file src="Frends.Web\FrendsTaskMetadata.json" target="FrendsTaskMetadata.json"/>
+	  <file src="..\Frends.Web\bin\Release\Frends.Web.dll" target="lib\net452\Frends.Web.dll" />
```

Comment:
This will probably fail our buildserver build of the nuget package. 
You can set the  BasePath parameter for the nuget.exe pack command to specify a base directory when you build it locally.